reference/widgets.v3.yml

```diff
@@ -1754,7 +1754,7 @@ components:
                 items:
                   type: object
                   title: widgetSchemaSetting_Base
-                  description: 'For examples of each type of setting, see [Page Builder > Schema Settings](/stencil-docs/page-builder/schema-settings/alignment) in Theme Docs.'
+                  description: 'For examples of each type of setting, see [Page Builder > Schema Settings](/stencil-docs/page-builder/schema-settings#alignment) in Theme Docs.'
```

Comment:
This link doesn't work.

Reply:
Agree with Traci, Found the same problem

Reply:
fixed